tests/testthat/test-get_environment.R

```diff
@@ -0,0 +1,32 @@
+## save current settings so that we can reset later
+threshold <- logger::log_threshold()
+appender  <- logger::log_appender()
+layout    <- logger::log_layout()
+on.exit({
+  ## reset logger settings
+  logger::log_threshold(threshold)
+  logger::log_layout(layout)
+  logger::log_appender(appender)
+})
+
+logger::log_appender(logger::appender_stdout)
+logger::log_threshold(logger::FATAL)
+logger::log_layout(logger::layout_simple)
```

Comment:
shame you can't [with_envvar()](https://withr.r-lib.org/reference/with_envvar.html) or  [with_options()](https://withr.r-lib.org/reference/with_options.html) here

Reply:
Probably can set up a helper, but that's for when I start refactoring tests to also test logging.

I'm imagining something along the lines of 


```
helper_suppress_logs()

test_that("test_1" {expect_foo()})

test_that("test 2", {
  expect_log(
    x <- foo(),
    debug = "foo!"
  )
})